tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -1,14 +1,32 @@
 [
+  {
+    "IntegrationName": "AwsSdk",
```

Comment:
It's kinda weird we never specified this here, but we need it to make sure the include/exclude works for regenerating package versions. Doesn't _need_ to be in this PR but meh